server/handlers/component_handler.go

```diff
@@ -84,14 +84,14 @@ func (h *Handler) GetAllComponents(rw http.ResponseWriter, r *http.Request) {
 
 // swagger:route GET /api/meshmodel/model GetMeshmodelModels idGetMeshmodelModels
 // Handle GET request for getting all meshmodel models. The component type/model name should be lowercase like "kubernetes", "istio"
-// responses:
 // ?version={version}
 // ?order={field} orders on the passed field
 // ?category={category of model}
```

Comment:
What is the behavior in the absence of a value? Is there a default value?
Is this API greedy? Are partial matches supported?


Please answer these questions across each operand.

Reply:
Good. And now for these answers on every other operand.

Reply:
The same is true for all operands

Reply:
Great. Clarify this in your docs. Consumers of these APIs shouldn't be left to guess each querystring option's behavior.